public/content/developers/docs/data-and-analytics/tool-descriptions.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Tool Descriptions
+description: A summary of the top tools and products in crypto data
+lang: en
+---
+
+## Introduction {#Introduction}
+
+Below you can find a description of some of the top data tools in the crypto space, as well as how to get started with them.
+
+## Block explorers {#block-explorers}
+
+Many [Block Explorers](/developers/docs/data-and-analytics/block-explorers/) offer [RESTful](https://www.wikipedia.org/wiki/Representational_state_transfer) [API](https://www.wikipedia.org/wiki/API) gateways that will provide developers visibility into real-time data on blocks, transactions, miners, accounts, and other on-chain activity.
+
+Developers can then process and transform this data to give their users unique insights and interactions with the [blockchain](/glossary/#blockchain). For example, [Etherscan](https://etherscan.io) provides execution and consensus data for every 12s slot.
```

Comment:
As mentioned above; this would be the third place we talk about block explorers....  Glad we didn't place a list here, but I would also not link to Etherscan. We should direct to our [block-explorers page](https://ethereum.org/en/developers/docs/data-and-analytics/block-explorers/)

Would probably ditch the "blockchain" glossary link here as well... feels out-of-place for this deep in the docs.